### PR TITLE
ts 3.9rc

### PR DESCRIPTION
**Pull request by chrisnojima**
- [x] faster: clean tsc takes 147 -> 130 on this laptop
- [x] catches more stuff

### CODE REVIEW

File: shared/chat/conversation/input-area/normal/platform-input.desktop.tsx [hunk 1]
```diff
@@ -314,7 +314,7 @@ class _PlatformInput extends React.Component<PlatformInputPropsInternal, State>
               onClick={this._inputFocus}
               selectable={true}
             >
-              *bold*, _italics_, `code`, >quote, @user, @team, #channel
+              {`*bold*, _italics_, \`code\`, >quote, @user, @team, #channel`}
```
Comment (chrisnojima) [inline shared/chat/conversation/input-area/normal/platform-input.desktop.tsx:317]: `>` is disallowed in jsx now

File: shared/chat/conversation/messages/account-payment/pending-background/index.native.tsx [hunk 2]
```diff
@@ -29,7 +29,7 @@ class PendingBackground extends React.Component<Props, State> {
       <>
         <Kb.NativeAnimated.Image
           resizeMode="repeat"
-          source={Styles.isDarkMode ? darkPatternImage : lightPatternImage}
+          source={Styles.isDarkMode() ? darkPatternImage : lightPatternImage}
```
Comment (chrisnojima) [inline shared/chat/conversation/messages/account-payment/pending-background/index.native.tsx:32]: this is a function! (more below)

File: shared/common-adapters/list2.native.tsx [hunk 3]
```diff
@@ -26,7 +26,7 @@ class List2<T> extends PureComponent<Props<T>> {
         return {index, length: itemHeight, offset: itemHeight * index}
       }
       case 'variable':
-        return {index, ...this.props.itemHeight.getItemLayout(index, data ? data[index] : undefined)}
```
Comment (chrisnojima) [inline shared/common-adapters/list2.native.tsx:29]: tsc now catches when you override a spread (more below)

File: shared/login/recover-password/explain-device/index.tsx [hunk 4]
```diff
@@ -41,7 +41,7 @@ const ExplainDevice = (props: Props) => {
             ) : (
               <Kb.Text type="Body">Settings</Kb.Text>
             )}
-            <Kb.Text type="Body">> Your account, and change your</Kb.Text>
```
Comment (chrisnojima) [inline shared/login/recover-password/explain-device/index.tsx:44]: incorrect!

Comment (cjb) [reply shared/login/recover-password/explain-device/index.tsx:44]: Shouldn't there be a &gt; though? "Settings > Your account"

Comment (chrisnojima) [reply shared/login/recover-password/explain-device/index.tsx:44]: Oh alrite I'll put it back

Comment (chrisnojima) [reply shared/login/recover-password/explain-device/index.tsx:44]: @cjb done

**Pull request merged**